src/migration.rs

```diff
@@ -183,6 +209,19 @@ where
         let store = self.client.get_or_create_store(&self.store_name).await?;
         let existing_models = self.get_existing_versions().await?;
         let max_existing_model = existing_models.iter().max();
+        // For the sake of MigrationFns, can we assume max_existing model is the currently active
+        // model?
+        let mut curr_model_id = if let Some(&version) = max_existing_model {
+            let id = self
+                .get_authorization_model_id(version)
+                .await?
+                .expect("Should get id of an existing model");
+            Some(id)
+        } else {
+            None
+        };
+        // TODO: get it for the first migration that's being run
```

Comment:
Probably a bit tricky and we don't need it for the upcoming migration in lakekeeper. So I'd suggest to open an issue and handle it in a separate PR.